R/derive_var_disposition_status.R

```diff
@@ -110,6 +115,150 @@ derive_disposition_status <- function(dataset,
                                       format_new_var = format_eoxxstt_default,
                                       filter_ds,
                                       subject_keys = vars(STUDYID, USUBJID)) {
+  deprecate_warn("0.6.0", "derive_disposition_status()", "derive_var_disposition_status()")
+  derive_var_disposition_status(dataset = dataset,
+                                dataset_ds = dataset_ds,
+                                new_var = !!enquo(new_var),
+                                status_var = !!enquo(status_var),
+                                format_new_var = format_new_var,
+                                filter_ds = !!enquo(filter_ds),
+                                subject_keys = subject_keys)
+}
+
+#' Default Format for Disposition Status
+#'
+#' Define a function to map the disposition status.
+#'
+#' @param x the disposition variable used for the mapping (e.g. `DSDECOD`).
+#'
+#' @return A `character` vector derived based on the values given in `x`:
+#'  "COMPLETED" if `x` is "COMPLETED",
+#'  "DISCONTINUED" if `x` is not "COMPLETED" nor NA,
+#'  "ONGOING" otherwise.
+#'
+#' @author Samia Kabi
+#' @export
+#' @keywords user_utility adsl computation
+format_eoxxstt_default <- function(x) {
+  case_when(
```

Comment:
The use of x as an input seems a bit odd to me?  Should we have a more descriptive name for the input>

Reply:
I agree that `x` is not a meaningful name. Thus we should change it. However, I am not sure if we should do it now or create a separate issue for the next release.

@thomas-neitmann , @bms63 , what do you think?

Reply:
I say lets make a separate issue for next release.  I'll do that

Reply:
Thanks

Reply:
#911